KyleHouston/dummy-game.js

```diff
@@ -0,0 +1,29 @@
+var MemoryGame = (function() {
```

Comment:
Looks good! Only issue I had was that card face images aren't displaying properly, which I've addressed farther down.